app/scripts/postinstall.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I didn't find any install errors. I don't see a need for the postinstall script

Reply:
Done 👍